src/requirements/WalletActivity/hooks/ContractDeployProvidedValue.tsx

```diff
@@ -0,0 +1,20 @@
+import { HStack, Text, useColorModeValue } from "@chakra-ui/react"
+import RequirementChainIndicator from "components/[guild]/Requirements/components/RequirementChainIndicator"
+import { ProvidedValueDisplayProps } from "requirements"
+
+const ContractDeployProvidedValue = ({ requirement }: ProvidedValueDisplayProps) => {
```

Comment:
Why are these components in hooks folder? We should rename the folders to providedValue imo (and make a separate folder when there're real hooks too)

Reply:
Originally, these were hooks instead of components, similar to how the rewards have their own hooks. Ive forgotten to update these, thanks!